datasets/nq_open/README.md

```diff
@@ -0,0 +1,195 @@
+---
+annotations_creators:
+- expert-generated
+language_creators:
+- other
+languages:
+- en
+licenses:
+- cc-by-sa-3.0
+multilinguality:
+- monolingual
+size_categories:
+- 10K<n<100K
+source_datasets:
+- 'extended|other-Natural Questions '
+task_categories:
+- question-answering
+task_ids:
+- open-domain-qa
+---
+
+# Dataset Card for nq_open
+
+## Table of Contents
+- [Dataset Card for nq_open](#dataset-card-for-nq_open)
+  - [Table of Contents](#table-of-contents)
+  - [Dataset Description](#dataset-description)
+    - [Dataset Summary](#dataset-summary)
+    - [Supported Tasks and Leaderboards](#supported-tasks-and-leaderboards)
+    - [Languages](#languages)
+  - [Dataset Structure](#dataset-structure)
+    - [Data Instances](#data-instances)
+    - [Data Fields](#data-fields)
+    - [Data Splits](#data-splits)
+  - [Dataset Creation](#dataset-creation)
+    - [Curation Rationale](#curation-rationale)
+    - [Source Data](#source-data)
+      - [Initial Data Collection and Normalization](#initial-data-collection-and-normalization)
+      - [Who are the source language producers?](#who-are-the-source-language-producers)
+    - [Annotations](#annotations)
+      - [Annotation process](#annotation-process)
+      - [Who are the annotators?](#who-are-the-annotators)
+    - [Personal and Sensitive Information](#personal-and-sensitive-information)
+  - [Considerations for Using the Data](#considerations-for-using-the-data)
+    - [Social Impact of Dataset](#social-impact-of-dataset)
+    - [Discussion of Biases](#discussion-of-biases)
+    - [Other Known Limitations](#other-known-limitations)
+  - [Additional Information](#additional-information)
+    - [Dataset Curators](#dataset-curators)
+    - [Licensing Information](#licensing-information)
+    - [Citation Information](#citation-information)
+
+## Dataset Description
+
+- **Homepage:** https://efficientqa.github.io/
+- **Repository:** https://github.com/google-research-datasets/natural-questions/tree/master/nq_open
+- **Paper:** https://www.aclweb.org/anthology/P19-1612.pdf
+- **Leaderboard:** https://ai.google.com/research/NaturalQuestions/efficientqa
+- **Point of Contact:** efficientqa@googlegroups.com
+
+### Dataset Summary
+
+The NQ-Open task, introduced by Lee et.al. 2019,
+is an open domain question answering benchmark that is derived from Natural Questions.
+The goal is to predict an English answer string for an input English question.
+All questions can be answered using the contents of English Wikipedia.
+
+### Supported Tasks and Leaderboards
+
+Open Domain Question-Answering, 
+EfficientQA Leaderboard: https://ai.google.com/research/NaturalQuestions/efficientqa
+
+### Languages
+
+English
+
+## Dataset Structure
+
+### Data Instances
+
+{
+      "question": datasets.Value("string"),
+      "answer": datasets.Sequence(datasets.Value("string")),
+}
```

Comment:
This field is intended to provide actual examples from the dataset in json format, such as:
```suggestion
{
    "question": "names of the metropolitan municipalities in south africa",
    "answer": [
        "Mangaung Metropolitan Municipality",
        "Nelson Mandela Bay Metropolitan Municipality",
        "eThekwini Metropolitan Municipality",
        "City of Tshwane Metropolitan Municipality",
        "City of Johannesburg Metropolitan Municipality",
        "Buffalo City Metropolitan Municipality",
        "City of Ekurhuleni Metropolitan Municipality"
    ]
}
```